libyul/backends/evm/EVMDialect.cpp

```diff
@@ -139,6 +139,25 @@ map<YulString, BuiltinFunctionForEVM> createBuiltins(langutil::EVMVersion _evmVe
 			Expression const& arg = _call.arguments.front();
 			_assembly.appendLinkerSymbol(std::get<Literal>(arg).value.str());
 		}));
+
+		builtins.emplace(createFunction(
+			"initfreemptr",
+			1,
+			0,
+			SideEffects{false, false, false, false, true},
+			{ true },
```

Comment:
Does this need to be true? If you don't set it to true, we can re-use the constant in other places.

Reply:
Oh wait a second, I think I misunderstood this here! I thought the argument is the location of the free memory pointer but the value is "hidden" inside the builtin function and is computed through the stack evader.